src/components/Button/deletebutton.js

```diff
@@ -0,0 +1,9 @@
+import StyledIcon from './styledicon';
+
+export default function DeleteButton({onClick}) {
+	return (
+		<StyledIcon onClick={onClick}>
```

Comment:
You dont need onClick at this point anymore 🙂

Reply:
I had the problem that the onClick function wouldn't work unless it was passed down to the child like here. I don't know why it wouldn't work but I found this suggestion on StackOverflow.